EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c

```diff
@@ -7,10 +7,15 @@
 #include "KeyMotorData.h"
 #include "MotorDetailsData.h"
 #include "DriverControlData.h"
+#include "MotorFaultsData.h"
+#include "BatteryFaultsData.h"
 
 #define KEY_MOTOR_LENGTH (43)
 #define MOTOR_DETAILS_LENGTH (69)
 #define DRIVER_CONTROLS_LENGTH (9)
+#define MOTOR_FAULTS_LENGTH (9)
+#define BATTERY_FAULTS_LENGTH (3)
```

Comment:
I wasn't really sure about this, since I was basing it off of `KEY_MOTOR_LENGTH` being 4 bytes shorter than on the protocol and figured there was 4 bytes of metadata?